docs/tutorials/projects/helmet.md

```diff
@@ -0,0 +1,455 @@
+
+1. Navigate to the **Services** subtab of your machine's **Config** tab.
+
+2. Click **Create service**.
+   Start typing `yolo` and select **vision / yolov8** from the registry options.
+   Click **Add module**.
+   You can find more details about this module in [its readme](https://github.com/viam-labs/YOLOv8).
+
+3. Give your vision service a name, for example `yolo`, and click **Create**.
+
+4. In the **Attributes** box of your new vision service, paste the following JSON:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "model_location": "keremberke/yolov8n-hard-hat-detection"
+   }
+   ```
+
+   This tells the vision service where to look for [the hard hat detection model](https://huggingface.co/keremberke/yolov8s-hard-hat-detection) we are using for this tutorial.
+
+   Your vision service config should now resemble the following:
+
+   {{<imgproc src="/tutorials/helmet/model-location.png" resize="x1100" declaredimensions=true alt="The vision service configured in the Viam app per the instructions." >}}
+
+5. Click **Save config** at the bottom of the screen.
+
+### Configure the `objectfilter` module
+
+The physical camera is working and the vision service is set up.
+Now you will pull them together with the [`objectfilter`](https://app.viam.com/module/felixreichenbach/object-filter) {{< glossary_tooltip term_id="module" text="module" >}}.
+This module takes a vision service (in this case, your hard hat detector) and applies it to your webcam feed.
+It outputs a stream with bounding boxes around the hard hats (and people without hard hats) in your camera's view so that you can see the detector working.
+This module also filters the output so that later, when you configure data management, you can save only the images that contain people without hard hats.
+
+1. Navigate to the **Components** subtab of your machine's **Config** tab.
+
+2. Click **Create component**.
+   Start typing `objectfilter` and select **camera / objectfilter** from the results.
+
+3. Name your filtering camera something like `objectfilter-cam` and click **Create**.
+
+4. Paste the following into the attributes box:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "filter_data": true,
+     "camera": "my_webcam",
+     "vision_services": ["yolo"],
+     "labels": ["Hardhat", "NO-Hardhat"],
+     "confidence": 0.5,
+     "display_boxes": true
+   }
+   ```
+
+   If you named your detector something other than "yolo," edit the `detector_name` line accordingly.
+   You can also edit the confidence threshold.
+   If you change it to `0.2` for example, you will see detections of over 20% confidence.
+
+   Setting the `filter_data` attribute to `true` means that later, when you configure data capture on this camera, only images that contain one or more of the labels will be captured.
```

Comment:
```suggestion
   Setting the `filter_data` attribute to `true` means that later, when you configure data capture on this camera, only images that have one or more of the labels will be captured and sent to the cloud.
   For testing purposes, leave both labels in the array for now.
   We will remove the `"Hardhat"` label from the configuration later so that the camera only captures and saves images when someone **isn't** wearing a hard hat.
```

I made a note here first asking why we were doing this because it would capture all the images. So good to explain it's for testing